wgsl/index.bs

```diff
@@ -13552,6 +13557,42 @@ Note: The result is not mathematically meaningful when `abs(e)` &ge; 1.
     <td>Returns the dot product of `e1` and `e2`.
 </table>
 
+### `dot4U8Packed` ### {#dot4U8Packed-builtin}
+<table class='data builtin'>
+  <tr algorithm="dot4U8Packed">
+    <td style="width:10%">Overload
+    <td class="nowrap"><xmp highlight=rust>
+@const @must_use fn dot4U8Packed(e1: u32,
+                                 e2: u32) -> u32
+</xmp>
+  <tr>
+    <td>Description
+    <td> `e1` and `e2` are interpreted as vectors with four 8-bit unsigned integer components.
+    Return the unsigned integer dot product of these two vectors.
+    Can only be used when
+    [=language-extension/packed_4x8_integer_dot_product|packed_4x8_integer_dot_product language extension=] is
+    enabled.
```

Comment:
Language extensions should be treated like core functionality except for the table. So this can be removed (also from the other functions added for this feature).